wandb/sdk/service/service.py

```diff
@@ -93,63 +71,39 @@ def _grpc_launch_server(self) -> Optional[int]:
             # Add coverage collection if needed
             if os.environ.get("COVERAGE_RCFILE"):
                 exec_cmd_list += ["coverage", "run", "-m"]
+            service_args = [
+                "wandb",
+                "service",
+                "--port-filename",
+                fname,
+                "--pid",
+                pid_str,
+                "--debug",
+            ]
+            service_args.append("--serve-sock")
+            # service_args.append("--serve-grpc")
```

Comment:
Should this be uncommented? We could add an `assertion` in the the `WandbServer` that we need the arguments be mutually exclusive (until we add support for both at same time)

Reply:
in progress.. currently the code you are looking at only works with sockets

Reply:
ok... got you!

Reply:
It is now:
```
            if self._use_grpc:
                service_args.append("--serve-grpc")
            else:
                service_args.append("--serve-sock")
```